ci/mac_ci_setup.sh

```diff
@@ -7,6 +7,9 @@
 # a list of pre-installed tools in the macOS image.
 
 export HOMEBREW_NO_AUTO_UPDATE=1
+HOMEBREW_RETRY_ATTEMPTS=10
+HOMEBREW_RETRY_INTERVAL=1
```

Comment:
possibly setting this to 3 would be better -  so the retries are spread over 30 seconds

Reply:
Yeah agreed, though this seems like an improvement so let's try this?